opm/simulators/flow/Main.hpp

```diff
@@ -329,17 +376,22 @@ namespace Opm
             }
             catch (const std::exception& e) {
                 if ( mpiRank == 0 )
+#ifdef OPM_FLOW_MAIN
                     std::cerr << "Exception received: " << e.what() << ". Try '--help' for a usage description.\n";
+#else
+                    Opm::Parameters::printUsage<PreTypeTag>(PreProblem::helpPreamble(
+                            argc, const_cast<const char**>(argv)), e.what());
+#endif
 #if HAVE_MPI
                 MPI_Finalize();
 #endif
                 return 1;
             }
-
+#ifdef OPM_FLOW_MAIN
```

Comment:
I think it would be an improvement fif we would always print the banner. The only reason not to do this would be compile problems.

Reply:
Agree, and there seems to be no compile problems by removing the `#ifdef`..

Reply:
Ok, then let's change it?

Reply:
Done with https://github.com/OPM/opm-simulators/pull/2523/commits/ee84e8bc2b20d75e135d605bc61e04b7c52bddb3